domains/u.json

```diff
@@ -0,0 +1,11 @@
+{
+        "owner": {
+           "username": "ImShyMike",
+           "discord": "582648583635992622"
+        },
+    
+        "record": {
+            "TXT": "https://shymike.is-a.dev"
```

Comment:
TXT records alone do nothing, did you mean URL?

Reply:
I did :kekw: